src/components/UnstyledLink/UnstyledLink.tsx

```diff
@@ -7,9 +7,10 @@ import {useLink, LinkLikeComponentProps} from '../../utilities/link';
 // but the props explorer isn't smart enough to work that out
 export interface UnstyledLinkProps extends LinkLikeComponentProps {}
 
+// This does have a display name, but the linting has a bug in it
+// https://github.com/yannickcr/eslint-plugin-react/issues/2324
+// eslint-disable-next-line react/display-name
```

Comment:
Is this needed anymore?

Reply:
Yep, still needed. We're using eslint-plugin-eslint-comments which contains rules than trigger a warning when we've got a disable line that doesn't do anything, so if this was unneeded it would be flagged.

This issue has been [resolved](https://github.com/yannickcr/eslint-plugin-react/pull/2399), and is in v7.15.0 but eslint-plugin-shopify depends up on v7.14.3 exactly.

On eslint-plugin-shopify's master branch that dependency has been bumped, so the next time theres a release of eslint-plugin-shopify this can be fixed up.

Reply:
Ok! Found it a bit strange that the eslint-disable-next-line was moved.

Reply:
There was a version bumb to eslint-plugin-react but not to the version we need - Now it raises a warning on both the React.memo and React.forwardRef - both of which are spurious.

Tbh there's no value in wrapping this in a memo anyway so I'm planning to remove that in a later PR.